src/components/bike/BikeList.vue

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+/* global Bike */
+
+import BikeCard from './BikeCard.vue'
+
+import { defineComponent, type PropType } from 'vue'
+
+export default defineComponent({
+  name: 'BikeList',
+  components: {
+    BikeCard
+  },
+  props: {
+    items: {
+      type: Array as PropType<Bike[]>,
+      default: () => []
+    }
+  },
+  data: () => ({})
+})
+</script>
+
+<template>
+  <div class="bike-list grid md:grid-cols-2 lg:grid-cols-3 gap-y-4 md:gap-6">
+    <template v-for="(item, i) in items" :key="`bike-${i}`">
```

Comment:
```suggestion
    <template v-for="item in items" :key="`bike-${i}`">
```

Reply:
Well, the `i` parameter is being used to create a unique key for the `<template>` block. I think it is not safe to remove it.

https://eslint.vuejs.org/rules/require-v-for-key.html